### ISSUE TITLE
Such critical App and NO Unit test?

### ISSUE THREAD

**Issue by orlovskyjavaprofi** [austrianredcross/stopp-corona-android#139]
Hello , where all unit tests, this really critical to write unit tests!

**Comment by mitsuhiko** [comment]
You're not wrong. This is something that generally help would be appreciated.

**Comment by falkorichter** [comment]
There are some unit tests: https://github.com/austrianredcross/stopp-corona-android/tree/release_2.0.3/app/src/test/java/at/roteskreuz/stopcorona but you are right, there could be more

**Comment by MJJacobs01** [comment]
Good afternoon @falkorichter,

Is this issue closed?

If not can I work on it?

**Comment by falkorichter** [comment]
Write all the unit tests! The app is obsolete, but if ahead.

**Comment by MJJacobs01** [comment]
Good morning,

Yes, but we might need to revive it in future, so better we test it now and
now that everything is working as expected. 😀

Thanks a lot
